bin/vitamin.js

```diff
@@ -68,7 +68,7 @@ const createCompiler = (webpackConfig, message, options) => {
     const compiler = webpack(webpackConfig);
     if (process.stdout.isTTY) {
         const bar = new ProgressBar(
-            `${chalk.blue(`\uD83D\uDD50 Building ${message}...`)} :percent [:bar]`,
+            `${chalk.blue(`\uD83D\uDD50  Building ${message}...`)} :percent [:bar]`,
```

Comment:
Because it was adjacent to the UTF8 symbol before.I think it's because the clock symbol takes two character spaces but I'm not sure.